termwiz/src/surface/line/line.rs

```diff
@@ -138,7 +138,7 @@ impl Line {
         let mut cells = Vec::new();
 
         for sub in Graphemes::new(s) {
-            let cell = Cell::new_grapheme(sub, attrs.clone(), unicode_version);
+            let cell = Cell::new_grapheme(sub, attrs.clone(), unicode_version.clone());
```

Comment:
This is also a hot code path, should probably pass `&unicode_version` here as well